packages/cubejs-sqlite-driver/package.json

```diff
@@ -14,7 +14,8 @@
   "main": "driver/SqliteDriver.js",
   "typings": "driver/index.d.ts",
   "scripts": {
-    "lint": "eslint **/*.js"
+    "lint": "eslint **/*.js",
+    "test": "jest ./**/*.test.js"
```

Comment:
Let's rename (or just add an alias) `unit` so this test could run in CI unit tests with `lerna run unit` - that would be useful.